sagemaker-pipelines/tabular/tensorflow2-california-housing-sagemaker-pipelines-deploy-endpoint/tensorflow2-california-housing-sagemaker-pipelines-deploy-endpoint.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    If Endpoint exist, then Endpoint will be updated with new Endpoint Config.
</pre><p>Grammar - add <code>s</code>:  <code>If Endpoint exists</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2865/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Fixed